tardis/util/base.py

```diff
@@ -539,3 +540,25 @@ def convert_abundances_format(fname, delimiter=r"\s+"):
     # Assign header row
     df.columns = [nucname.name(i) for i in range(1, df.shape[1] + 1)]
     return df
+
+
+def check_simulation_env():
+    """
+    Checks for the Simulation Environment
+
+    Returns
+    -------
+    True : if the environment is IPython Based
+    False : if the environment is Terminal or anything else
+    """
+    try:
+        shell = get_ipython().__class__.__name__
+    except NameError:
+        return False
+
+    if shell == "ZMQInteractiveShell":
+        return True
+    elif shell == "TerminalInteractiveShell":
+        return False
+    else:
```

Comment:
when does this else statement happen?

Reply:
The following needs to be considered for the conditional check:
- If we are in IPython Kernel environment, ie Jupyter Notebook, Jupyter Lab, etc, then we are in the `ZMQInteractiveShell`. Please check [this](https://ipython.readthedocs.io/en/stable/config/options/kernel.html).
- If we are in the Terminal IPython environment, ie IPython Kernel running in Terminal, then we are in the `TerminalInteractiveShell`. Please check [this](https://ipython.readthedocs.io/en/stable/config/options/terminal.html).
- If we are in any other environment, ie Google Colab or Normal Shell based Terminal, etc, then we are in the else condition. Hence, the main `else` condition allows us to print the values as a str in #1632

Reply:
![image](https://user-images.githubusercontent.com/66117751/122001004-48e6b100-cdcd-11eb-8059-c55f502cd2b5.png)
*This is Jupyter Notebook*

![image](https://user-images.githubusercontent.com/66117751/122000866-1b9a0300-cdcd-11eb-9c86-8490b3b535f4.png)
*This is IPython in terminal*

![image](https://user-images.githubusercontent.com/66117751/122000914-2eacd300-cdcd-11eb-913d-0dc742a8d27e.png)
*This is normal Python from command line